### ISSUE TITLE
Fixed Reference Points - Potential editorial updates to the SC text

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#1461]
Editorial suggestions:
- Does the the SC need to use the "web page or set of web pages" phrase? In either case, it's "web content" or even just "content".
- There are 2 independent clauses in the SC without a separating coma. 
- SC verb is plural and should be singular. When singular nouns are joined by "or", the verb should be singular. Should read "...formatting or platform changes."
- Suggest using "page break" rather than "pagebreak" which is an HTML element. (Same goes for the definition of "Virtual page".)

With the editorial changes, suggest the SC reads: _When the content is an electronic publication with page break locators, a mechanism is available to navigate to each locator, and each locator maintains its place in the flow of content, even when the formatting or platform changes._

Clarification needed:
- What is the scope of "platform", as there's no definition of what is a platform where electronic documents on the web are concerned. Concern is that this SC my be scoping in that the web-based document author is somehow responsible for how the pagination compares to non-web platforms (that may be marked up using a different language, such as MOBI) or even printed documents. The understanding didn't clarify the scope.

**Comment by alastc** [comment]
I've created a PR with those changes in #1461 

I'm not sure about the platform question. Generally we try to avoid defining dictionary terms like platform.

However, it leads to a question for @DavidMacDonald and the ePub folks: Can a page break locator loose it's place in the document flow, even on different formats/platforms? How would an author control that?

Perhaps we can just leave that last bit off?

**Comment by alastc** [comment]
Hi Mary Jo, 

We've got a new version which I think addresses (or removes!) everything you'd commented about:
https://raw.githack.com/w3c/wcag/wcag22-fixed-ref-points-updates-2021-jan/understanding/22/fixed-reference-points.html

I've been through the changes I made in #1461, and I think it's defunct now. Either we'll agree the new version, or defer the SC.
